client/stylesheets/controls.styl

```diff
@@ -210,7 +202,7 @@
     width 100%
     max-height 56px
     line-height 16px
-    overflow hidden
+    overflow scroll
```

Comment:
For browsers that show fixed scroll bars, the horizontal scroll bar eats up
a lot of vertical space, so I'd rather hide it.

On Mon, May 18, 2015 at 2:17 PM, David Manthey <david.manthey@kitware.com>
wrote:

> Yes.
>
> On Mon, May 18, 2015 at 2:17 PM, Jeffrey Baumes <notifications@github.com>
> wrote:
>
>> In client/stylesheets/controls.styl
>> <https://github.com/XDATA-Year-3/geoapp/pull/94#discussion_r30533313>:
>>
>> > @@ -210,7 +202,7 @@
>> >      width 100%
>> >      max-height 56px
>> >      line-height 16px
>> > -    overflow hidden
>> > +    overflow scroll
>>
>> Where does overflow-x come into play? Really long words?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/XDATA-Year-3/geoapp/pull/94/files#r30533313>.
>>
>
>
>
> --
> David Manthey
> R&D Engineer
> Kitware Inc.
> (518) 881-4439
>



-- 
David Manthey
R&D Engineer
Kitware Inc.
(518) 881-4439